ui/js/models/resultsets_store.js

```diff
@@ -14,6 +14,9 @@ treeherder.factory('ThResultSetStore', [
 
         var $log = new ThLog("ThResultSetStore");
 
+        // indexOf doesn't work on objects so we need to map thPlatformMap to an array
+        var platformArray = _.map(thPlatformMap, function(val, idx) { return idx; });
```

Comment:
Yeah, as you'd mentioned, I would have this as another entity in the ``values.js`` file.  Just have it do this same calculation, but cache it like:

    if (_.isUndefined(platformArray)) {
        platformArray = _.map(thPlatformMap, function(val, idx) { return idx; });
    }
    return platformArray;

That way it only does that work once.